Common/src/gRPC/ListApplicationsRequestExt.cs

```diff
@@ -48,29 +57,41 @@ public static class ListApplicationsRequestExt
     }
   }
 
-  public static Expression<Func<TaskData, bool>> ToApplicationFilter(this ListApplicationsRequest.Types.Filter filter)
+  /// <summary>
+  ///   Converts gRPC message filters into an <see cref="Expression" /> that represents the filter conditions
+  /// </summary>
+  /// <param name="filters">The gPRC filters</param>
+  /// <returns>
+  ///   The <see cref="Expression" /> that represents the filter conditions
+  /// </returns>
+  /// <exception cref="ArgumentOutOfRangeException">the given message is not recognized</exception>
+  public static Expression<Func<TaskData, bool>> ToApplicationFilter(this Filters filters)
   {
     var predicate = PredicateBuilder.New<TaskData>();
```

Comment:
the default value given by this function is false?

Reply:
yes